tests/data/test_loader.py

```
@@ -0,0 +1,54 @@
+    """Test loading Excel data from file path."""
+    file_path = os.path.join(TEST_FILES_DIR, "example_input.xlsx")
+    df = load_excel_data(file_path)
+    
+    assert isinstance(df, pd.DataFrame)
+    assert df.shape[1] > 0  # Should have at least one column
+    assert not df.empty
+
+
+def test_load_excel_invalid_file():
+    """Test loading Excel data from an invalid file."""
+    with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)